### ISSUE TITLE
rpn_bbox_weight is missing while computing rpn l1 loss?

### ISSUE THREAD

**Issue by hdjsjyl** [TuSimple/simpledet#237]
The following is the official code for computing rpn l1 loss:

# regression loss
        reg_loss = X.smooth_l1(
            (bbox_delta - bbox_target),
            scalar=3.0,
            name="rpn_reg_l1"
        )
        reg_loss = bbox_weight * reg_loss
        reg_loss = X.loss(
            reg_loss,
            grad_scale=1.0 / (batch_image * image_anchor) * scale_loss_shift,
            name="rpn_reg_loss"
        )

When I use the following code to draw the network structure to a pdf:
a = mx.viz.plot_network(sym)
    a.render('net_train')

The network structure pdf version doesn't have bbox_weight, which is corresponding the position of red arrow in the following picture:
![rpn_bbox_weight](https://user-images.githubusercontent.com/7590139/65359603-a7fc9f00-dbc2-11e9-860a-3569c904e041.png)

**Comment by RogerChern** [comment]
The weight is used as shown in `checkpoint.json`
```
    {
      "op": "elemwise_mul",
      "name": "_mul0",
      "inputs": [[446, 0, 0], [462, 0, 0]]
    },
```

The viz tool in MXNet is written 3 or 4 years ago. We recommend you try Netron for visualization.
